forest/src/cli/net_cmd.rs

```diff
@@ -14,21 +14,17 @@ use forest_rpc_client::net_ops::*;
 #[derive(Debug, StructOpt)]
 pub enum NetCommands {
     /// Lists `libp2p` swarm listener addresses
-    #[structopt(about = "List listen addresses")]
     Listen,
     /// Lists `libp2p` swarm peers
-    #[structopt(about = "Print peers")]
     Peers,
-    /// Connects to a peer
-    #[structopt(about = "Connect to a peer by its peer ID and multiaddresses")]
+    /// Connects to a peer by its peer ID and multi-addresses
     Connect {
-        #[structopt(about = "Multiaddr (with /p2p/ protocol)")]
+        /// Multi-address (with /P2P/ protocol)
```

Comment:
I would use more `/p2p/` here (so in quotes ``).

Reply:
👍